src/main/java/seedu/multitasky/logic/commands/DeleteByFindCommand.java

```diff
@@ -19,13 +20,17 @@
     public static final String MESSAGE_NO_ENTRIES = "No entries found! Please try again with different keywords";
 
     public static final String MESSAGE_MULTIPLE_ENTRIES = "More than one entry found! \n"
-            + "Use " + COMMAND_WORD + " [" + String.join(" | ", CliSyntax.PREFIX_EVENT.toString(),
-            CliSyntax.PREFIX_DEADLINE.toString(), CliSyntax.PREFIX_FLOATINGTASK.toString()) + "]"
-            + " INDEX to specify which entry to delete.";
+                                                          + "Use " + COMMAND_WORD + " ["
+                                                          + String.join(" | ",
+                                                                        CliSyntax.PREFIX_EVENT.toString(),
+                                                                        CliSyntax.PREFIX_DEADLINE.toString(),
+                                                                        CliSyntax.PREFIX_FLOATINGTASK.toString())
+                                                          + "]"
+                                                          + " INDEX to specify which entry to delete.";
 
```

Comment:
haha, its alright. just leave it as it is. i got the neat copy on my branch still, i can just use my version when i merge into v0.4